server/src/main/java/org/apache/iotdb/db/mpp/plan/parser/ASTVisitor.java

```diff
@@ -2217,4 +2219,19 @@ public Statement visitFlush(IoTDBSqlParser.FlushContext ctx) {
     flushStatement.setStorageGroups(storageGroups);
     return flushStatement;
   }
+
+  // show region
+
+  @Override
+  public Statement visitShowRegion(IoTDBSqlParser.ShowRegionContext ctx) {
+    ShowRegionStatement showRegionStatement = new ShowRegionStatement();
+    if (ctx.DATA() != null) {
+      showRegionStatement.setRegionType(TConsensusGroupType.DataRegion);
+    } else if (ctx.SCHEMA() != null) {
+      showRegionStatement.setRegionType(TConsensusGroupType.SchemaRegion);
+    } else {
+      showRegionStatement.setRegionType(TConsensusGroupType.PartitionRegion);
```

Comment:
Set to Null? Maybe we will add a show partition region in the future.

Reply:
fixed~